specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -13360,284 +14719,1310 @@
+            "$ref": "#/definitions/ReplicationAgentDetails"
+          },
+          "readOnly": true
+        },
+        "reprotectAgents": {
+          "description": "The list of reprotect agents.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReprotectAgentDetails"
+          },
+          "readOnly": true
+        },
+        "marsAgents": {
+          "description": "The list of Mars agents.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MarsAgentDetails"
+          },
+          "readOnly": true
+        },
+        "dras": {
+          "description": "The list of DRAs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DraDetails"
+          },
+          "readOnly": true
+        },
+        "agentDetails": {
+          "description": "The list of agent details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AgentDetails"
+          },
+          "readOnly": true
+        }
+      },
+      "x-ms-discriminator-value": "InMageRcm"
+    },
+    "InMageRcmFailbackEventDetails": {
+      "description": "Event details for InMageRcmFailback provider.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EventProviderSpecificDetails"
+        }
+      ],
+      "properties": {
+        "protectedItemName": {
+          "description": "The protected item name.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-discriminator-value": "InMageRcmFailback"
+    },
+    "InMageRcmFailbackMobilityAgentDetails": {
+      "description": "InMageRcmFailback mobility agent details.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "description": "The agent version.",
+          "type": "string",
+          "readOnly": true
+        },
+        "latestVersion": {
+          "description": "The latest agent version available.",
+          "type": "string",
+          "readOnly": true
+        },
+        "driverVersion": {
+          "description": "The driver version.",
+          "type": "string",
+          "readOnly": true
+        },
+        "latestUpgradableVersionWithoutReboot": {
+          "description": "The latest upgradeable version available without reboot.",
+          "type": "string",
+          "readOnly": true
+        },
+        "agentVersionExpiryDate": {
+          "format": "date-time",
+          "description": "The agent version expiry date.",
+          "type": "string",
+          "readOnly": true
+        },
+        "driverVersionExpiryDate": {
+          "format": "date-time",
+          "description": "The driver version expiry date.",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastHeartbeatUtc": {
+          "format": "date-time",
+          "description": "The time of the last heartbeat recieved from the agent.",
```

Comment:
received

Reply:
done

---
In reply to: [588092366](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r588092366) [](ancestors = 588092366)